### ISSUE TITLE
when used as loss, SSIM/MSSIM should be negative 

### ISSUE THREAD

**Issue by lelikchern** [photosynthesis-team/photosynthesis.metrics#37]
SSIM and MSSIM currently compute the similarity index which is between 0 and 1, if arguments are given within correct ranges.

However, usually people _minimize_ losses in Pytorch. So, unless they use the _negative_ SSIM and MSSIM, they will end up with images that are dissimilar to their targets.  I would suggest writing this word of advice somewhere in SSIM/MSSIM documentation or examples, just in case, for beginners, etc., e.g.:

**If you want to train a neural network to produce images that are similar to your target, you should use SSIM or MSSIM losses with a negative sign.**

**Comment by denproc** [comment]
The `SSIMLoss` and `MultiScaleSSIMLoss` were changes to return values for minimisation at #55 .
